packages/components/psammead-media-player/src/Guidance/index.jsx

```diff
@@ -12,16 +12,32 @@ const GUIDANCE_BACKGROUND = 'rgba(34, 34, 34, 0.75)';
 const GuidanceWrapper = styled.div`
   ${({ service }) => getSansRegular(service)}
   ${GEL_LONG_PRIMER};
-
   width: 100%;
   height: 100%;
   position: absolute;
-  background-color: ${GUIDANCE_BACKGROUND};
   border: 0.0625rem solid transparent;
   color: ${C_WHITE};
+  background-color: ${({ guidanceMessage }) =>
```

Comment:
Will this still be set when `guidanceMessage` is not defined?

```
${guidanceMessage ? `background-color: ${GUIDANCE_BACKGROUND};` : ``}
```

Reply:
@DenisHdz no it won't be set

Reply:
It seems that it will be an invalid CSS rule, so the browser will ignore it. https://github.com/styled-components/styled-components/issues/402

If you prefer to revert it to keep it more readable, up to you, but we should always check the browser skips it and we are not generating broken/invalid css :)

Reply:
@DenisHdz I've adjusted it